PyPlaque/utils/fp_readout_object.py

```diff
@@ -78,23 +79,58 @@ def get_major_minor_axis_length(self):
         return self.plaque_object_properties.axis_major_length, \
                 self.plaque_object_properties.axis_minor_length
     
+    def get_custom_eccentricity(self,):
+        # Calculate the image moments
+        y_indices, x_indices = np.nonzero(binary_image)
```

Comment:
Here where do we get the binary_image variable from ? Could raise an error here. Can you please check ?